src/main/java/org/mule/tools/apikit/model/MuleConfigBuilder.java

```diff
@@ -79,4 +78,12 @@ public static MuleConfig fromStream(InputStream input) throws Exception {
   private static boolean elementIsApikitRouter(Element element) {
     return element.getNamespace().getPrefix().equals("apikit") && element.getName().equals("router");
   }
+
+  static SAXBuilder getSaxBuilder() {
+    SAXBuilder builder = new SAXBuilder();
+    builder.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
+    builder.setFeature("http://xml.org/sax/features/external-general-entities", false);
+    builder.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
```

Comment:
Shoud we make this configurable via system properties?

Reply:
Added a javadoc with explanation